examples_tests/61.BlitFilterTest/blit/formats/r11f_g11f_b10f.glsl

```diff
@@ -0,0 +1,59 @@
+#ifndef _NBL_GLSL_BLIT_R11FG11FB10F_INCLUDED_
+#define _NBL_GLSL_BLIT_R11FG11FB10F_INCLUDED_
+
+uint nbl_glsl_encode_ufloat(in float _f32, in uint mantissaMask, in uint mantissaBits)
+{
+	const uint f32 = floatBitsToUint(_f32);
+
+	if ((f32 & 0x80000000u) != 0u) // negative numbers converts to 0 (represented by all zeroes)
+		return 0;
+
+	const uint expMask = 0x1fu << mantissaBits;
+
+	const int exp = int(((f32 >> 23) & 0xffu) - 127);
+	const uint mantissa = f32 & 0x7fffffu;
```

Comment:
we have `nbl_glsl_ieee754` functions for this

Reply:
I don't think there is one for mantissa. Should I make it?

Reply:
yes please

Reply:
Resolved.